### ISSUE TITLE
RPi keeps rebooting every 5 sec

### ISSUE THREAD

**Issue by cjdcordeiro** [PiSupply/PiJuice#397]
Hi,

I'm having issues with the PiJuice Hat. 
On full charge (static blue LED), when I boot the RPi it just starts the bootstrap and goes straight into a loop.

I've tried plugging the RPi directly to power and monitor the battery with pijuice_cli, and everything seems normal. 

I've tried resetting everything to default and also upgrading firmware to v1.3.
Nothing changed. 

I'm running Raspbian Buster Lite on a RPi 3

**Comment by tvoverbeek** [comment]
With power to the RPi and running pijuice_cli check on the 'System Events' page
what the state is of 'No power:'.  If it is enabled and set to e.g. SYS_FUNC_REBOOT
you get what you see: a reboot loop when no external power is applied.

**Comment by cjdcordeiro** [comment]
@tvoverbeek no it's not enabled. All system events are disabled

**Comment by tvoverbeek** [comment]
Any peripherals connected to the RPi? E.g a usb-disk spin-up at boot could draw too much power and causing a shutdown.
If not, it might be a defective unit. Take contact with your supplier  for a replacement.

**Comment by cjdcordeiro** [comment]
nope. Just had a Pimoroni Blinkt. But removed it and the problem presisted

**Comment by shawaj** [comment]
@cjdcordeiro did you manage to get this sorted now? Can we close this issue?

**Comment by cjdcordeiro** [comment]
I didn't. but you can close it anyway
